RELEASING.md

```diff
@@ -1,48 +1,63 @@
-# Releasing OpenTelemetry Packages (for maintainers only)
-This document explains how to publish all OT modules at version x.y.z. Ensure that you’re following semver when choosing a version number.
+# Release instructions
+
+## Preparing a new major or minor release
+
+* Merge a pull request to `main` updating the `CHANGELOG.md`.
+  * The heading for the release should include the release version but not the release date, e.g.
+    `## Version 1.9.0 (unreleased)`.
+  * Use `.github/scripts/draft-change-log-entries.sh` as a starting point for writing the change
+    log.
+* Run the [Prepare release branch workflow](https://github.com/trask/repository-template/actions/workflows/prepare-release-branch.yml).
+  * Press the "Run workflow" button, and leave the default branch `main` selected.
+* Review and merge the two pull requests that it creates
+  (one is targeted to the release branch and one is targeted to `main`).
+
+## Preparing a new patch release
+
+* Backport pull request(s) to the release branch.
+  * Run the [Backport workflow](https://github.com/trask/repository-template/actions/workflows/backport.yml).
```

Comment:
Curious as to why this is called `backport.yml`?

Reply:
It backports a PR from the main branch to a release branch, so that PR can be included in a patch release